documentation/book/assembly-using-the-kafka-bridge.adoc

```diff
@@ -0,0 +1,27 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+[id='assembly-using-the-kafka-bridge-{context}']
+= Using the {ProductName} Kafka Bridge
+
+This chapter provides an overview of the {ProductName} Kafka Bridge and helps you get started with using the API. To view the API documentation including example requests and responses, see LINK **{ProductName} Kafka Bridge API resources**. For information on how to configure the Kafka Bridge after deployment, see LINK **{ProductName} Kafka Bridge Configuration**.
```

Comment:
I would say REST API in the first sentence at least, not repeating every time API is mentioned.

Reply:
Good point.